modules/oe_whitelabel_paragraphs/oe_whitelabel_paragraphs.module

```diff
@@ -0,0 +1,529 @@
+ */
+function oe_whitelabel_paragraphs_theme_suggestions_field_alter(array &$suggestions, array $variables): void {
+  $element = $variables['element'];
+
+  // Do not output field labels and wrapping markup for paragraph fields.
+  if (isset($element['#entity_type']) && $element['#entity_type'] === 'paragraph') {
+    // Prepend the new suggestion to the list. This will put it right after the
+    // default field template. By doing this we allow to override single
+    // fields, while keeping all the rest markup-free.
+    array_unshift($suggestions, 'field__bare');
+  }
+}
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_whitelabel_paragraphs_preprocess_paragraph__oe_links_block(array &$variables): void {
+  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+  $paragraph = $variables['paragraph'];
+  $variables['orientation'] = $paragraph->get('oe_bt_links_block_orientation')->value;
+  $variables['background'] = $paragraph->get('oe_bt_links_block_background')->value;
+  $variables['title'] = $paragraph->get('field_oe_text')->value ?? '';
+
+  foreach (Element::children($variables['content']['field_oe_links']) as $index) {
+    $variables['links'][] = [
+      'label' => $variables['content']['field_oe_links'][$index]['#title'],
+      'path' => $variables['content']['field_oe_links'][$index]['#url']->toString(),
+    ];
+  }
+}
+
+/**
+ * Implements hook_preprocess_paragraph() for oe_social_media_follow paragraph.
+ */
+function oe_whitelabel_preprocess_paragraph__oe_social_media_follow(array &$variables): void {
+  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+  $paragraph = $variables['paragraph'];
+  $variables['orientation'] = $paragraph->get('field_oe_social_media_variant')->value;
+  $variables['background'] = $paragraph->get('oe_bt_links_block_background')->value;
+  $variables['title'] = $paragraph->get('field_oe_title')->value ?? '';
+  $links = $paragraph->get('field_oe_social_media_links')->getValue();
+  $variables['links'] = [];
+  foreach ($links as $key => $link) {
+    $variables['links'][$key]['icon_position'] = 'before';
+    $variables['links'][$key]['icon']['path'] = $variables['bcl_icon_path'];
+    $variables['links'][$key]['icon']['name'] = $link['link_type'];
+    $variables['links'][$key]['label'] = $link['title'];
+    $variables['links'][$key]['path'] = Url::fromUri($link['uri'])->toString();
+  }
+  if (!$paragraph->get('field_oe_social_media_see_more')->isEmpty()) {
+    $other_link = $paragraph->get('field_oe_social_media_see_more')
+      ->first()
+      ->getValue();
+    $variables['links'][] = [
+      'label' => $other_link['title'],
+      'path' => Url::fromUri($other_link['uri'])->toString(),
+    ];
+  }
+}
+
+/**
+ * Implements hook_preprocess_paragraph__oe_accordion().
+ */
+function oe_whitelabel_preprocess_paragraph__oe_accordion(array &$variables): void {
+  // Massage data to be compliant with OE Bootstrap Theme accordion component
```

Comment:
double check all references to bootstrap pls

Reply:
The "OE Bootstrap Theme" is correct here, because this is where the pattern is.
Only change "component" to "pattern" to be more precise.